doc/Command-Reference.md

```diff
@@ -3559,6 +3587,65 @@ This command displays the details of Rx & Tx priority-flow-control (pfc) for all
   root@sonic:~# sonic-clear pfccounters
   ```
 
+**show pfc asymmetric**
```

Comment:
This looks like duplication, we already have Asymmetric PFC mode shown as part of ```show interfaces status``` command (it is also described in ```Command-Reference.md``` doc under the ```show interfaces status``` section):
```
root@sonic:/home/admin# show interfaces status
      Interface            Lanes    Speed    MTU    Alias             Vlan    Oper    Admin             Type    Asym PFC
---------------  ---------------  -------  -----  -------  ---------------  ------  -------  ---------------  ----------
      Ethernet0          0,1,2,3      40G   9100     etp1           routed    down     down   QSFP+ or later         off
      Ethernet4          4,5,6,7      40G   9100     etp2            trunk      up       up   QSFP+ or later         off
...
```

Reply:
@volodymyrsamotiy Yes, this is true. But there is no harm.